rcgen/Cargo.toml

```diff
@@ -31,7 +31,8 @@ x509-parser = { workspace = true, features = ["verify"], optional = true }
 zeroize = { version = "1.2", optional = true }
 
 [features]
-default = ["pem", "ring"]
+default = ["crypto", "pem", "ring"]
+crypto = []
```

Comment:
Original comment by @djc: https://github.com/rustls/rcgen/pull/207#pullrequestreview-1817771475

> This doesn't seem to entirely make sense to me. I don't see a clear use case for wanting random but not crypto?

Noted. I made it only `crypto` in this case.

Reply:
the ring and aws-lc-rs features should probably also depend on `crypto`. It makes no sense to enable ring/aws-lc-rs and not enable `crypto`.

Reply:
That's a good point. I can do that.